plugins/woocommerce/src/Blocks/BlockTypes/ProductFilters.php

```diff
@@ -35,17 +35,18 @@ protected function enqueue_data( array $attributes = array() ) {
 		global $pagenow;
 		parent::enqueue_data( $attributes );
 
-		$this->asset_data_registry->add( 'isBlockTheme', wp_is_block_theme() );
-		$this->asset_data_registry->add( 'isProductArchive', is_shop() || is_product_taxonomy() );
-		$this->asset_data_registry->add( 'isSiteEditor', 'site-editor.php' === $pagenow );
```

Comment:
isSiteEditor and isWidgetEditor do not appear to be used, is it ok to remove them?

Reply:
They were used in the past, but logic using them got removed I think. It's safe to remove them for me, too.